src/AutoRest.CSharp/Mgmt/Models/MgmtRestOperation.cs

```diff
@@ -185,10 +186,10 @@ public MgmtRestOperation(MgmtRestOperation other, string nameOverride, CSharpTyp
             var finalSchema = Operation.LongRunning?.FinalResponse.BodyType;
             if (finalSchema is null)
                 return null;
-
+            (InputType finalType, bool isNullable) = finalSchema is InputNullableType nullableType ? (nullableType.Type, true) : (finalSchema, false);
             try
             {
-                return finalSchema is InputModelType inputModel ? MgmtContext.Library.ResolveModel(inputModel) : MgmtContext.TypeFactory.CreateType(finalSchema);
+                return finalType is InputModelType inputModel ? MgmtContext.Library.ResolveModel(inputModel).WithNullable(isNullable) : MgmtContext.TypeFactory.CreateType(finalSchema);
```

Comment:
can we just return `MgmtContext.TypeFactory.CreateType(finalSchema)`?

Reply:
yeah we should not do the same thing in multiple places.